xwiki-platform-core/xwiki-platform-mail/xwiki-platform-mail-ui/src/main/resources/Mail/SendMailStatus.xml

```diff
@@ -76,32 +76,36 @@
 {{translation key="admin.emailstatus.hint" /}}
 
 ##
-## Livetable #
+## Live Data #
 ##
 #set($columns = ['date', 'batchId', 'type', 'recipients', 'state', 'error', 'actions'])
-#set($columnsProperties = {
-    'date' : { 'type' : 'date' },
-    'batchId' : { 'type' : 'text' },
-    'type' : { 'type' : 'text' },
-    'recipients' : { 'type' : 'text' , 'html' : 'true' },
-    'state' : { 'type' : 'text', 'size' : 6 },
-    'error' : { 'type' : 'text' },
-    'actions' : { 'type' : 'text', 'html' : 'true', 'filterable' : 'false', 'sortable' : 'false' }
-})
 #if ($xcontext.isMainWiki())
-  #set($discard = $columns.add(5, 'wiki'))
-  #set($discard = $columnsProperties.put('wiki', { 'type' : 'text', 'size' : 6 }))
+  #set ($discard = $columns.add(5, 'wiki'))
 #end
-#set($options = {
-   'resultPage': 'Mail.SendMailStatusJSON',
-   'translationPrefix' : 'Mail.SendMailStatus_livetable.',
-   'tagCloud' : false,
-   'selectedColumn' : 'date',
-   'defaultOrder' : 'desc',
-   'outputOnlyHtml': true
+#set ($liveDataConfig = {
+  'meta': {
+    'propertyDescriptors': [
+      { 'id': 'date', 'displayer': 'text' },
+      { 'id': 'batchId', 'displayer': 'text' },
+      { 'id': 'type', 'displayer': 'text' },
+      { 'id': 'recipients', 'displayer': 'html' },
+      { 'id': 'state', 'displayer': 'text' },
+      { 'id': 'wiki', 'displayer': 'text' },
+      { 'id': 'error', 'displayer': 'text' },
```

Comment:
``text`` displayer is used by default so there's no need to specify it. Moreover, there's no need to specify a property descriptor if you don't have any (custom) configuration for it, so most of the property descriptors above can be omitted.